docs/conf.in.py

```diff
@@ -264,8 +271,22 @@ def skip(app, what, name, obj, would_skip, options):
     return would_skip
 
 
+def escape_vertical_bars(app, what, name, obj, options, lines: List[str]):
+    # Note - Latex docs also contains Vertical Bar, so we
+    # apply this filtering only to specific classes.
+    if name in [
```

Comment:
Could put `name` as an `ESCAPE_VERTICAL_BAR_WHITELIST` const outside the function.

Reply:
Hi Wei, Thanks for reviewing the PR. Did you mean to put list of classes outside `escape_vertical_bars` function as:-

```
ESCAPE_VERTICAL_BAR_CLASSES: List[str] = [
    "open3d.data.RedwoodIndoorLivingRoom1",
    "open3d.data.RedwoodIndoorLivingRoom2",
    "open3d.data.RedwoodIndoorOffice1",
    "open3d.data.RedwoodIndoorOffice2",
]


def escape_vertical_bars(app, what, name, obj, options, lines: List[str]):
    # Note - Latex docs also contain Vertical Bar, so we
    # apply this filtering only to specific classes.
    if name in ESCAPE_VERTICAL_BAR_CLASSES:
        for index in range(len(lines)):
            lines[index] = lines[index].replace("|", "\|")
```

Reply:
Yes, this looks much cleaner. Thank you!